app/imports/collections/comments.js

```diff
@@ -0,0 +1,19 @@
+import { Meteor } from 'meteor/meteor';
+
+class CommentsCollection extends Mongo.Collection {
+  insert(doc, callback) {
+    doc.createdAt = doc.createdAt || new Date();
+    doc.userId = Meteor.userId();
+    doc.year = doc.year || new Date().getFullYear();
```

Comment:
the year? now that i look at this code i think the year should come from the public settings.

what do you think?